source/index.txt

```diff
@@ -9,8 +9,10 @@ Start Developing with MongoDB
 .. meta:: 
    :keywords: r, elixir, prisma, mongoose
    
-Connect your application to your database with one of the official MongoDB 
-libraries.
+Connect your application to your `MongoDB Atlas database
+<https://www.mongodb.com/docs/atlas>`__ or `self-hosted MongoDB
+database <https://www.mongodb.com/docs/manual/installation>`__ with
```

Comment:
Issue:
From [Links and Cross-References](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/) in the style guide:
> Strive to be clear about where exactly a link will take a reader.

and

> To minimize reader disruption, aim to keep users on the page they're currently reading as much as possible.

From [Terminology](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u):
> using ... Use by using, or recast the sentence. Don't use using (alone) or with. 

Suggestion:

I think something like the following could address all the aforementioned issues:

"You can connect your application to your MongoDB Atlas deployment or a self-hosted MongoDB cluster by using one of the official MongoDB libraries. To learn more about Atlas, see :atlas:`What is MongoDB Atlas? </>`. To learn how to download and install MongoDB, see :manual:`Install MongoDB </installation>`."

Reply:
Thank you! I made this change with one small difference (specify that the install page is for self-hosted)